apps/opik-backend/src/test/java/com/comet/opik/api/resources/utils/StatsUtils.java

```diff
@@ -318,4 +319,50 @@ public static int bigDecimalComparator(BigDecimal v1, BigDecimal v2) {
         return strippedV1.toBigInteger().compareTo(strippedV2.toBigInteger());
     }
 
+    public static int closeToEpsilonComparator(Object v1, Object v2) {
```

Comment:
Most likely, we don't need this. There are assertJ assertions such as isCloseTo that allow different types of approximations e.g: based on percentage, boundaries etc.

Reply:
Actually I can't find a way to make it using assertj's `isCloseTo` due to the recursive comparison. I can try to simplify `closeToEpsilonComparator`'s logic if you think it's worth it.